inventory/group_vars/all.yml

```diff
@@ -82,6 +83,11 @@ openshift_cluster_content:
     namespace: labs-ci-cd
     tags:
     - nexus
+    post_step:
```

Comment:
this should be `post_steps` according to: https://github.com/redhat-cop/casl-ansible/blob/master/roles/openshift-applier/tasks/process-content.yml#L31

Reply:
yes, that needs to be `post_steps` - will fix (I believe that was a change post PR with multiple things happening in parallel)